cypress/fixtures/customers.json

```diff
@@ -0,0 +1,82 @@
+{
+  "customers": [
+    {
+      "id": 1,
+      "name": "Jacobs Co",
+      "employees": 99,
+      "industry": "Logistics",
+      "contactInfo": null,
+      "address": {
+        "street": "988 Kimberly Fort Apt. 921",
+        "city": "Lake Tracy",
+        "state": "Connecticut",
+        "zipCode": "07115",
+        "country": "United States of America"
+      },
+      "size": "Small"
+    },
+    {
+      "id": 2,
+      "name": "Kilback Co",
+      "employees": 100,
+      "industry": "Retail",
+      "contactInfo": {
+        "name": "Daija",
+        "email": "Daija_Gislason93@gmail.com"
+      },
+      "address": {
+        "street": "5099 Murray Inlet",
+        "city": "South Tiffany",
+        "state": "Kentucky",
+        "zipCode": "08496",
+        "country": "United States of America"
+      },
+      "size": "Medium"
+    },
+    {
+      "id": 3,
+      "name": "Parisian Co",
+      "employees": 999,
+      "industry": "Technology",
+      "contactInfo": {
+        "name": "Alysson",
+        "email": "Alysson.Lang@hotmail.com"
+      },
+      "address": {
+        "street": "43247 Bennett Keys Apt. 999",
+        "city": "New Paulside",
+        "state": "Connecticut",
+        "zipCode": "87855",
+        "country": "United States of America"
+      },
+      "size": "Medium"
+    },
+    {
+      "id": 4,
+      "name": "Wilderman Co",
+      "employees": 1000,
+      "industry": "HR",
+      "contactInfo": {
+        "name": "Brando",
+        "email": "Brando_Kozey48@gmail.com"
+      },
+      "address": {
+        "street": "8643 Jackson Wall",
+        "city": "Lake Davidstad",
+        "state": "Minnesota",
+        "zipCode": "29481",
+        "country": "United States of America"
+      },
+      "size": "Enterprise"
+    },
+    {
+      "id": 5,
+      "name": "Nitzsche, Williamson and Hartmann",
+      "employees": 2710,
+      "industry": "Logistics",
+      "contactInfo": null,
+      "address": null,
+      "size": "Enterprise"
+    }
+  ]
+}
```

Comment:
```suggestion
}

```

Recomenda-se uma linha em branco no final de cada arquivo, para que o GitHub não exiba o ícone de `No newline at end of file`, conforme comentado no último encontro.